docs/modules/ROOT/pages/preface.adoc

```diff
@@ -11,7 +11,7 @@ linear fashion or you can skip sections if something does not interest you.
 == About the documentation
 The Spring Cloud Task reference guide is available in https://docs.spring.io/spring-cloud-task/docs/current/reference[html].
 The latest copy is available at
-https://docs.spring.io/spring-cloud-task/docs/current-SNAPSHOT/reference/html/.
+https://docs.spring.io/spring-cloud-task/reference/
```

Comment:
Thank you for your contribution.  Can you please add a period to the end of the sentence?